### PR TITLE
Donor can view profile page

### PR DESCRIPTION
**Pull request by swardbladpaulo**
[API: Donor can view a profile page](https://www.pivotaltracker.com/story/show/176728781)
### User Story
```
As a donor
In order to be able to view my profile information
I would like to be able to access a personalized profile page.
```
## Changes proposed in this pull request:
* Creating userController
* Modifying routes.rb
* Creating a migration
* Creating a test for donor_can_view_profile_page
## What have we learned working on this feature:
* Working more with binding.pry
* Working with the tests
## Screenshots (Spec)
Screenshot from 2021-01-30 18-34-08

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -48,6 +48,10 @@
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.integer "role", default: 0
+    t.string "adress"
+    t.string "zipcode"
```
Comment (Ehugo2000) [inline db/schema.rb]: integer?

File: app/controllers/api/user_controller.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class Api::UserController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_user, only: :show
+
```
Comment (Ehugo2000) [inline app/controllers/api/user_controller.rb:4]: format!

**Review by Ehugo2000 — COMMENTED** [review]
some small thingys

File: db/schema.rb (hunk 1 above)
Comment (swardbladpaulo) [reply db/schema.rb]: done.

File: app/controllers/api/user_controller.rb (hunk 2 above)
Comment (swardbladpaulo) [reply app/controllers/api/user_controller.rb:4]: done.

**Pull request merged**